examples/speed_tester/speed_tester.c

```diff
@@ -424,16 +430,18 @@ nf_setup(struct onvm_nf_info *nf_info) {
                         onvm_ft_fill_key(&key, pkt);
                         pkt->hash.rss = onvm_softrss(&key);
 
-                        onvm_nflib_return_pkt(nf_info, pkt);
+                        /* Add packet to batch */
+                        pkts[i - 1] = pkt;
```

Comment:
This doesn't look pretty, can  you change it to `pkts[i] = pkt;`? The i++ increment should be after this statement